### PR TITLE
Switch docs

### PR DESCRIPTION
**Pull request by moonso**
Switch from using deprecated gitbook to mkdocs and host docs under clinical genomics

### CODE REVIEW

**Comment by moonso** [comment]
Hi @northwestwitch , I have just switched documentation from using gitbook to mkdocs like we have for most other tools. The old way of doing this was broken. Could you please have a quick look?

File: Pipfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "d434516416c9015943442988fe91cc7b6c27cd4f3e6a96c867ea00a43b6ff1f6"
```
Comment (northwestwitch) [inline Pipfile.lock:4]: I never used pipenv so there's not much I can check here, I trust you with this code

File: README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ to get started!
 
 
 [unix]: http://en.wikipedia.org/wiki/Unix_philosophy
-[docs]: http://www.chanjo.co/en/latest/
+[docs]: http://www.clinicalgenomics.se/chanjo/
```
Comment (northwestwitch) [inline README.md:77]: 👍

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (northwestwitch) [inline mkdocs.yml:1]: Looks fine

File: Pipfile.lock (hunk 1 above)
Comment (moonso) [reply Pipfile.lock:4]: It's pipenv who does this automatically :)

Comment (northwestwitch) [reply Pipfile.lock:4]: Good, otherwise it would be crazy updating this document!

**Pull request merged**